README.rst

```diff
@@ -6,6 +6,8 @@ General Knight Lab Admin Commands for sequence processing in Qiita.
 
 This is an Admin only plugin
 
+If `pip install -e .` fails, try using `pip install -e . --use-pep517` as a substitute.
```

Comment:
Do you know what changed and how can it be fixed?

Reply:
After spending a few hours looking into this, it seems the best chance of fixing it is to replace setup.py with the new pyproject.toml. I have something that passes syntax but it seems the new specification doesn't like discovering dependencies metapool and mg-scripts. Right now it seems fairly open-ended. It might be better to wrap up the other items and resolve this after the release. What do you think?

Reply:
Thank you for checking; what's special about those dependencies? Basically, why it complains about `metapool` and `mg-scripts` but not `qiita-files`?

Reply:
`error: Multiple top-level packages discovered in a flat-layout: ['foo', 'test1', 'qp_klp'].
      
      To avoid accidental inclusion of unwanted files or directories,
      setuptools will not proceed with this build.
      
      If you are trying to create a single distribution with multiple packages
      on purpose, you should not rely on automatic discovery.
      Instead, consider the following options:
      
      1. set up custom discovery (`find` directive with `include` or `exclude`)
      2. use a `src-layout`
      3. explicitly set `py_modules` or `packages` with a list of names
      
      To find more information, look for "package discovery" on setuptools docs.`

Based on the error message above, it seems like I need to do more than just put those projects inside the 'install_requires' listing. The original translation was done by https://github.com/gvalkov/setuptools-py2cfg although the results needed substantial changes to even reach the message above. I don't think it's just metapool and mg-scripts but more defining dependencies in general. At this point I need to dig through the specs more, which can be time-consuming. I also don't know where foo and test1 came from, since they are not referenced anywhere in metapool, mg-scripts, or qp-klp build settings. Based on all that, my thinking is to close the loop on this PR and make removing the switch a post-release issue.

Reply:
Will this work? https://stackoverflow.com/a/72547402

Reply:
Resolved w/https://github.com/qiita-spots/qp-knight-lab-processing/pull/72